### DISCUSSION TITLE
6.1.0: match cache improvements and more

### DISCUSSION THREAD

**Discussion by jslater89** [jslater89/uspsa-result-viewer#28] [Announcements]
6.1.0 has a ton of new features, mostly involving new ways to view results and research the historical causes of rating changes.

## General changes

-  Add match cache search/management dialog on main page: add single matches, view matches, or remove matches from the cache.
-  Various match cache improvements.
-  Fix some match parsing issues.
-  Swap to 'this is a link' mouse pointer on a larger number of clickable areas.

## Ratings

-  Clicking on a stage name or graph segment in the shooter stats dialog takes you to the relevant match results.
-  Show estimated percentage in prediction error bars, rather than raw predictions.
-  Add button to view match results for matches in a ratings dataset.
-  Add a 'parse practiscore page source' option for adding matches to a rater dataset: follow the instructions in the help dialog to view page source, let Analyst do the rest.
-  Retain original member number, and link shooter names to uspsa.org shooter pages in various places.
-  Clicking the 'histogram' text in rater stats dialog switches to quartile view.
-  Rater stats dialog calculates stats for the currently-filtered shootrs only.
-  Improvements to 'processing scores' portion of loading screen.
-  Inverse place points series rater slightly simplified: you get 1 point for each person you beat

<hr /><em>This discussion was created from the release <a href='https://github.com/jslater89/uspsa-result-viewer/releases/tag/6.1.0'>6.1.0: match cache improvements and more</a>.</em>

**Comment by apnea1963** [comment]
Do you have a manual for using the app? I've seen what it can do but can't figure out the how. Thanks!

**Reply by jslater89** [reply]
Someday! I'm in the middle of a big rewrite of the data engine, and documentation would be a moving target at the moment.

**Comment by apnea1963** [comment]
Understood. Thanks for replying and for your work on this project. I’m looking forward to see what you do. Sent from my iPhoneOn Jan 26, 2024, at 19:13, jslater89 ***@***.***> wrote:﻿
Someday! I'm in the middle of a big rewrite of the data engine, and documentation would be a moving target at the moment.
